modules/xmpp/SdpConsistency.js

```diff
@@ -78,6 +38,9 @@ export default class SdpConsistency {
      *  in future calls to makeVideoPrimarySsrcsConsistent
      */
     setPrimarySsrc (primarySsrc) {
+        if (typeof primarySsrc !== 'number') {
```

Comment:
Also the JSDoc says it wants a number so it enforces number. I will add throws description as well